integritybackend/actions.py

```diff
@@ -426,9 +426,11 @@ def c2pa_proofmode(self, zip_path: str, org_config: dict, collection_id: str):
                     raise Exception(f"ZIP at {zip_path} has no content metadata file")
                 with zipf.open(meta_content_path) as meta_content_f:
                     meta_content = json.load(meta_content_f)["contentMetadata"]
-                    photographer_id = asset_helper.filename_safe(
-                        meta_content["private"]["signal"]["sourceName"]
-                    )
+                    source_name = meta_content.get("private", {}).get("signal", {}).get("sourceName")
```

Comment:
```suggestion
                    source_name = meta_content.get("private", {}).get("author", {}).get("name")
```

`sourceName` removed in favor of `author`.`name`

Reference:
https://github.com/starlinglab/integrity-preprocessor/commit/63db7ea4f5960399e7fdd6644a857a8b7d64e425#diff-b28dae66ef5c5cc1cf8999d5338ff8d6b74f2fdcabfa9081eb85252a9373be99L380